pkgs/development/python-modules/ansible-lint/default.nix

```diff
@@ -53,20 +51,14 @@ buildPythonPackage rec {
     "test_prerun_reqs_v1"
     "test_prerun_reqs_v2"
     # Assertion error with negative numbers; maybe requieres an ansible update?
-    "test_negative"
-    "test_example"
-    "test_playbook"
     "test_included_tasks"
-    "test_long_line"
 
```

Comment:
Any reason these tests below are in a separate block? It would be nice to have them aggregated by failure type (same exception, same root cause) and have a comment why they are failing.